community/cups-filters/build

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+./configure  \
+    --prefix=/usr \
+    --sysconfdir=/etc \
```

Comment:
Add `--sbindir=/usr/bin` (There's a file in `sbin`). :+1:

Reply:
Done!